metricbeat/module/dropwizard/collector/collector_integration_test.go

```diff
@@ -67,19 +71,14 @@ func TestFetch(t *testing.T) {
 	}
 	assert.Equal(t, hasTag, true)
 	assert.Equal(t, doesntHaveTag, true)
-	if !assert.NoError(t, err) {
-		t.FailNow()
-	}
 
 	t.Logf("%s/%s event: %+v", f.Module().Name(), f.Name(), events)
 }
 
 func TestData(t *testing.T) {
-	compose.EnsureUp(t, "dropwizard")
+	f := mbtest.NewReportingMetricSetV2Error(t, getConfig())
```

Comment:
Now that we have the new data generation, we could remove `TestData`?

Reply:
Absolutely